wandb/sdk/internal/sender.py

```diff
@@ -508,6 +523,11 @@ def _config_telemetry_update(self, config_dict: Dict[str, Any]) -> None:
         b = self._telemetry_obj.env.kaggle
         config_dict[wandb_key]["is_kaggle_kernel"] = b
 
+        b = self._telemetry_obj.feature.offline
+        config_dict[wandb_key]["is_offline"] = b
```

Comment:
Can you remove this, it is redundant.
The only reason we have the other fields is because they were there before we had the proper encoded telemetry data.